dir.targets

```diff
@@ -62,6 +62,15 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="GenerateRuntimeConfig" BeforeTargets="Build" Condition="$(OutputType) == 'Exe' ">
+    <ItemGroup>
+      <RuntimeConfigTemplate Include="$(ToolsDir)\xunit.console.netcore.runtimeconfig.json"/>
```

Comment:
Should we check in an extra config file so we aren't relying on this being around? That might be unnecessarily cautious.